.github/workflows/deploy.yaml

```
@@ -51,6 +51,13 @@ jobs:
           labels: ${{ steps.meta.outputs.labels }}
           platforms: linux/amd64
 
+      - name: Generate artifact attestation
+        uses: actions/attest-build-provenance@v3
+        with:
+          subject-name: ghcr.io/${{ github.repository }}
+          subject-digest: ${{ steps.push.outputs.digest }}
```

actionlint (1.7.8)

property "push" is not defined in object type {meta: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)